src/components/Button/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Button.css';
+
+export type ButtonProps = {
+  onClick: (e: React.MouseEvent) => void;
+  children: React.ReactNode;
+  disabled: boolean;
```

Comment:
лучше сделать необязательным 
```suggestion
  disabled?: boolean;
```